src/docs/product/accounts/sso/okta-sso/okta-scim.mdx

```diff
@@ -172,7 +166,7 @@ If the user is not part of a group with an organization-level role assigned, re-
 
 ## Known Issues
 
-- The Import Users feature is not currently supported because Sentry's SCIM API doesn't support the User `name` attribute fields: `firstName` and `lastName`at this time. For compatibility purposes, they're returned with `N/A` values. 
+- The Import Users feature is not currently supported because Sentry's SCIM API doesn't support the User `name` attribute fields: `firstName` and `lastName`at this time. For compatibility purposes, they're returned with `N/A` values.
```

Comment:
```suggestion
- The Import Users feature isn't currently supported because Sentry's SCIM API doesn't support the user `name` attribute fields: `firstName` and `lastName` at this time. For compatibility purposes, they're returned as `N/A` values.
```